Automattic/jetpack
# Migrate Block Patterns Registration to jetpack-mu-wpcom

## Proposed changes:


This PR migrates the Block Patterns feature (inserting wpcom Block Patterns to the editor) into `jetpack-mu-wpcom`, which was originally located in ETK ([editing-toolkit-plugin/block-patterns](https://github.com/Automattic/wp-calypso/tree/e4467c3/apps/editing-toolkit/editing-toolkit-plugin/block-patterns)). 

### Other information:


## Jetpack product discussion





## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Patch this PR by following the instructions https://github.com/Automattic/jetpack/pull/34162#issuecomment-1814111621
* Disable ETK 
	* on Simple site: By removing this filter https://github.com/Automattic/wp-calypso/blob/b81dab43af076713053372cf2ca8f32fb5a90a3d/apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php#L286-L296
	* on Atomic site: By uninstalling ETK
* Go to the Post Editor
* Open the block inserter
* Check for categories (e.g., Intro)
* Try inserting patterns

<img width="646" alt="Screenshot 2023-11-16 at 12 16 13" src="https://github.com/Automattic/jetpack/assets/5287479/687dbc58-c05c-468e-8876-19cd4c781cb9">
